.github/configure-san

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Runs ./configure with
+#     1. --disable-bootstrap
+#     2. ASAN
+
+# This should be run by github CI
+# However, running it locally (if it works on your PC) might be useful
+
+export GCC_LIBS=$(gcc -print-search-dirs | head -n1 | sed 's/install: //')
+export COMMON="-fsanitize=address"
+
+CONFIGURE=$(dirname $(dirname $0))/configure
+
+$CONFIGURE CFLAGS="$COMMON -B$GCC_LIBS" \
```

Comment:
Consider using `--with-build-config="bootstrap-asan"` (or whatever it is) instead, but not sure how to handle tests (some DejaGnu hackery may be needed there).

Reply:
`bootstrap-asan` doesn't work with `--disable-bootstrap`, as far as I can tell

Reply:
Ah, you're right, but I see you've figured it out ;)